src/components/SearchBar/SearchBar.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Could you please rewrite this test. Add test case with snapshot check as you have now, add test case to simulate change of input and check that state has been changed properly, add test case which will call prop 'onHandleSearchBy' of SearchBy component (`wrapper.find(SearchBy).prop('SearchBy')()`) and check that state has been changed properly